.github/workflows/pr-test.yml

```
@@ -172,4 +172,4 @@ jobs:
           readarray -t files_to_check <<< "$GIT_DIFF_FILES"
 
           export CONTENT_ROOT=$(pwd)/files
-          yarn filecheck "${files_to_check[@]}"
+          npm run filecheck -- "${files_to_check[@]}"
```

Unnecessary double dash:

```suggestion
          npm run filecheck "${files_to_check[@]}"
```